google-cloud-spanner/src/test/java/com/google/cloud/spanner/DatabaseClientImplTest.java

```diff
@@ -3009,6 +3013,10 @@ public void testDatabaseOrInstanceDoesNotExistOnCreate() {
               "Instance", SpannerExceptionFactory.INSTANCE_RESOURCE_TYPE, INSTANCE_NAME)
         };
     for (StatusRuntimeException exception : exceptions) {
+      if (isMultiplexedSessionsEnabled()) {
```

Comment:
I don't think you need the `if` statement. `createSessions` is never used by the regular session pool, so you can just always add the exception for that as well.

Reply:
Removed.